docs/components/connectors/introduction.md

```diff
@@ -2,125 +2,45 @@
-As a Self-Managed user, you can run it standalone, but often not directly reuse the logic in your existing Zeebe Client that you might already have.
-You can manually extract the job handler from the given job worker, but you also have to ensure that it is still working as expected afterward.
-
-In contrast, a Connector itself is environment-agnostic. There is a runtime environment for Camunda Platform 8 SaaS that can wrap and call this Connector. As the Connector developer, you don't have to worry about this as the runtime takes care of it if you developed the Connector using the Connector SDK.
-
-You can also run the exact same Connector (without any modification) in Camunda Platform 8 Self-Managed; either as a standalone job worker, as additional job handler in your existing Zeebe Client application, or together with other Connectors in one Zeebe Client application.
-This all comes with the Connector SDK, and there is no additional code necessary to get started. However, if you need a custom environment, the Connector SDK provides a guide and default helpers to do that.
-
-#### Focus
-
-A job worker is often a complete Zeebe Client application, dealing with environment tasks like handling variables in and out. The core logic of calling a defined URL is only part of the application.
-Plus, it handles Camunda Platform 8-specific APIs like the job worker API to handle variables, complete executions, and throw errors.
+A Connector consists of two parts:
 
-A Connector only consists of core business functionality. No environment tasks, no Camunda Platform 8 job worker-related code. You can run this from Camunda Platform 7 as well, if you have a runtime that takes care of this.
-The Connector only needs input variables and access to secrets so they can be used in defined input attributes.
+1. The programming code in Java to connect to the external system (for example, see the [Connector function](./connector-sdk.md#runtime-logic) for outbound Connectors.)
+2. The user interface to be used during modeling, which is provided using [Connector templates](./connector-templates.md).
 
-#### Context
+Users interacting with Connectors may only need to understand the configuration options exposed by the Connector template in the properties panel.
 
-Every job worker implementation defines on its own how to handle input data, validating and transforming it.
-There is no unified way of using secrets in a job worker implementation either, e.g. to replace placeholders in attributes with sensitive information only at runtime.
-Plus, there is no unified modeling experience for job workers. There can be an element template for the worker, but that template might look completely different for every job worker.
+Connectors are available in three categories - [out-of-the-box](./out-of-the-box-connectors/available-connectors-overview.md) , [template-only](./connector-templates.md), and custom via the [Connector SDK](./connector-sdk.md).
```

Comment:
If this is the first reference of "out-of-the-box", I would likely write as "**out-of-the-box (OOTB)**" to clarify usage of this acronym.